python_src/src/lamp_py/performance_manager/l1_rt_trips.py

```diff
@@ -93,15 +93,13 @@ def load_new_trip_data(db_manager: DatabaseManager) -> None:
         .distinct(
             TempEventCompare.service_date,
             TempEventCompare.route_id,
-            TempEventCompare.direction_id,
-            TempEventCompare.start_time,
-            TempEventCompare.vehicle_id,
+            TempEventCompare.trip_id,
         )
         .order_by(
             TempEventCompare.service_date,
             TempEventCompare.route_id,
-            TempEventCompare.direction_id,
-            TempEventCompare.start_time,
+            TempEventCompare.trip_id,
+            TempEventCompare.stop_sequence.desc(),
```

Comment:
The purpose of this query is to collect additional information fields ( vehicle_label, vehicle_consist) for unique trips that will be used for UPDATE/INSERT operations into our vehicle_trips table.

For this first stop-event in a trip, this information is frequently carried over from the last trip for the vehicle, so this query is collecting the latest stop-event values from these information fields to UPDATE/INSERT into our vehicle_trips table.

That is why the desc() ORDER is used, it would probably be helpful to add a comment to this effect above these calls.